.github/workflows/format.yml

```diff
@@ -0,0 +1,33 @@
+name: format
+
+on: push
+
+jobs:
+
+  java_format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2 # v2 minimum required
+      - uses: axel-op/googlejavaformat-action@v3
+        with:
+          args: "--skip-sorting-imports --replace"
+
+  objc_format:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2 # v2 minimum required
+      - name: Test
+        run: |
+          which clang-format || brew install clang-format
+          find . -name '*.m' -exec clang-format -i {} \;
```

Comment:
I'm not an expert with Linux commands but if i'm not mistaken this particular line should ideally change to `find . -name '*.m' -or -name '*.h' clang-format -i {} \;` to allow for it to analyse header (`.h`) files too

Reply:
Done. :)